<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ARIDE</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QTreeWidget" name="listObjects" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Project</string>
    </property>
    <addaction name="actionConfigure_Environments" />
    <addaction name="actionConfigure_Display_Sets" />
   </widget>
   <widget class="QMenu" name="menuControl" >
    <property name="title" >
     <string>Control</string>
    </property>
    <addaction name="actionRun" />
    <addaction name="actionStop" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="action_New" />
    <addaction name="actionOpen" />
    <addaction name="actionSave" />
    <addaction name="actionSave_As" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuControl" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>595</y>
     <width>669</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
  <action name="actionOpen" >
   <property name="text" >
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionConfigure_Environments" >
   <property name="text" >
    <string>Configure &amp;Environments</string>
   </property>
  </action>
  <action name="actionConfigure_Display_Sets" >
   <property name="text" >
    <string>Configure &amp;Display Sets</string>
   </property>
  </action>
  <action name="actionRun" >
   <property name="text" >
    <string>Run</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="action_New" >
   <property name="text" >
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionSave_As" >
   <property name="text" >
    <string>Save &amp;As</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
